sdk/math/wasm_helpers.rs

```diff
@@ -62,14 +63,28 @@ extern "C" {
 }
 
 #[macro_export]
-macro_rules! scale {
+macro_rules! decimal_ops {
     ($decimal:ident) => {
         ::paste::paste! {
             #[wasm_bindgen]
             #[allow(non_snake_case)]
             pub fn [<get $decimal Scale >] () -> BigInt {
                 BigInt::from($decimal::scale())
             }
+
+            #[wasm_bindgen]
+            #[allow(non_snake_case)]
+            pub fn [<get $decimal Denominator >] () -> BigInt {
+                BigInt::from(10u128.pow($decimal::scale() as u32))
```

Comment:
There is already denominator implemented in decimal: `BigInt::from($decimal::from_integer(1).get())`